docs/source/quickstart/99_FAQ.md

```diff
@@ -8,6 +8,9 @@ the playbook doesn't reload your environment, mostly because we're using the `lo
 
 In order to solve this issue, please logout and re-login your account.
 
+Alternatively, you can set `cifmw_libvirt_manager_user` to your USERNAME in the my_env
```

Comment:
hmmm I don't see how passing that option may work around the group issue?

Reply:
so this is how Zuul gets added - https://github.com/openstack-k8s-operators/ci-framework/blob/b7cfec873a2086ebce29924e49a57224e939ef7a/ci_framework/roles/libvirt_manager/tasks/virsh_checks.yml#L34 ....
https://github.com/openstack-k8s-operators/ci-framework/blob/b7cfec873a2086ebce29924e49a57224e939ef7a/ci_framework/roles/local_env_vm/files/firstboot.sh#LL26C1-L26C1 - alternative - but this needs to be done after the libvirt group is created

Reply:
the "zuul" user is for the local_env VM (and RDO CI env) - it should really be your user by default:
`cifmw_libvirt_manager_user: "{{ ansible_user | default(lookup('env', 'USER')) }}"`

So I'm a bit surprised, really...